### DISCUSSION TITLE
regEx search support

### DISCUSSION THREAD

**Discussion by andremoraes** [usinesoft/Cachalot#8] [Q&A] [answered]
I've compiled and used the samples and I want to use Cachalot to store pdf pages data do show tags matching a regex expression, does the sql or LINQ supports that? on the pdf, there's no mention of regEx support

can you do join, left join, and is there documentation of what it's supported on the SQL side? I'm more comfortable in writing SQL vs. LINQ.

ty

**Comment by usinesoft** [comment, marked as answer]
Hello, thank you for your interest,
The SQL documentation is available here: 
https://github.com/usinesoft/Cachalot/blob/master/Doc/SQLandLINQguide.pdf

There is no regex support for now. There is support for string operations StartsWith = LIKE pattern% EndsWith = LIKE %pattern and Contains = LIKE %pattern%. 
It also explains why classical JOIN can not be implemented in a distributed system and how to replace it.

**Reply by andremoraes** [reply]
ty
